db/models.py

```diff
@@ -50,3 +54,52 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return str(self.created_at)
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(MovieSession, on_delete=models.CASCADE)
+    order = models.ForeignKey("Order", on_delete=models.CASCADE)
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    def __str__(self) -> str:
+        return (f"{self.movie_session.movie.title}"
+                f" {self.movie_session.show_time} "
+                f"(row: {self.row}, seat: {self.seat})")
+
+    def clean(self) -> None:
+        cinema_hall = self.movie_session.cinema_hall
+        if self.row < 1 or self.row > cinema_hall.rows:
+            raise ValidationError(
+                {"row": [f"row number must be in available range: "
+                         f"(1, rows): (1, {cinema_hall.rows})"]})
+        if self.seat < 1 or self.seat > cinema_hall.seats_in_row:
+            raise ValidationError(
+                {"seat": [f"seat number must be in available range:"
+                          f" (1, seats_in_row): "
+                          f"(1, {cinema_hall.seats_in_row})"]})
+
+    def save(self, *args, **kwargs) -> None:
+        self.full_clean()
+        super().save(*args, **kwargs)
+
+    class Meta:
+        constraints = [models.UniqueConstraint(
+            fields=["row", "seat", "movie_session"],
+            name="unique")]
```

Comment:
The constraint name 'unique' is not descriptive. Consider using a more descriptive name, such as 'unique_ticket_per_session', to clarify the purpose of the constraint.